src/labelle/lib/devices/device_manager.py

```diff
@@ -82,8 +81,11 @@ def find_and_select_device(self, patterns: list[str] | None = None) -> UsbDevice
             LOG.debug(dev.device_info)
         dev = devices[0]
         if dev.is_supported:
-            msg = f"Recognized device as \
-                {SUPPORTED_PRODUCTS[SUPPORTED_DEVICE_ID(dev.id_product)]}"
+            foundDeviceConfig: DeviceConfig | None = get_device_config_by_id(
+                dev.id_product
+            )
+            if foundDeviceConfig is not None:
+                msg = f"Recognized device as {foundDeviceConfig.name}"
```

Comment:
What is `msg` if `found_device_config` is `None`?

Reply:
I was not sure how to proceed than. Just print nothing?
Is there a quick syntax to "add if not null, otherwise leave empty"

Reply:
Fixed